lib/internal/modules/esm/loader.js

```diff
@@ -614,5 +623,6 @@ function register(specifier, parentURL = undefined, options) {
 module.exports = {
   createModuleLoader,
```

Comment:
is this export still in use?

Reply:
Not anymore. Removed.

Reply:
Uh I guess not, it's used by ` test/es-module/test-esm-loader-modulemap`